castanet/counter.py

```diff
@@ -2,34 +2,28 @@
 from typing import List, Tuple, Dict, Optional
 # Import LibCST
 import libcst as cst
-import generate_trees as generator
-
+from castanet import generate_trees as generator
 import libcst.matchers as match
-    
-# TODO: Create function definitions related to all necessary user stories (should look somewhat like match_imports)
-# TODO: Modify annotations to have node key as type and key as necessary information
-
 
-def match_imports(cast_dict):
+def match_ifstatements(cast_dict):
     """A function for counting the number of if statements in a Python program."""
-    imports_dictionary = {}
+    ifstatements_dictionary = {}
     # Iterate through all of the Python files in a directory
     for file in cast_dict:
         # Find CASTs for each of these files
         cast = cast_dict[file]
-        # Determine number of import statements for each file
-        imports = match.findall(cast, match.Import())
-        imports_dictionary[file] = len(imports)
-
-    return imports_dictionary
+        # Determine number of if statements for each file
+        ifs = match.findall(cast, match.If())
+        ifstatements_dictionary[file] = len(ifs)
 
+    return ifstatements_dictionary
 
 if __name__ == "__main__":
```

Comment:
You can remove everything in the `if __name__ == "__main__":` function, as the program is not run alone (lines21-28)